src/ActivityDefinition.php

```diff
@@ -41,13 +41,9 @@ public function __construct() {
             $this->_fromArray($arg);
         }
 
-        foreach (
-            [
-                'name',
-                'description',
-                'extensions',
-            ] as $k
-        ) {
+        $props = ['name', 'description', 'extensions'];
+
+        foreach ($props as $k) {
```

Comment:
What is the reason for this change? Does PHP7 not handle the existing formatting?